token-lending/program/tests/fixtures/README.md

```diff
@@ -1,11 +1,16 @@
 # fixtures
 
-### SOL / USDC / SRM Aggregator Accounts
-
-<!--- @FIXME: replace with actual aggregator accounts --->
+### SOL / SRM Aggregator Accounts
 
 ```shell
 solana config set --url https://devnet.solana.com
-solana account 9u7fuQCJCX13HhU8DZoj4bwP3vWsDFVWkurNkv7JvhYh --output-file btc_usd_aggregator.bin
-solana account ELz3y7PCa6gp8GjD8TrXnggmS2bxsDrznrxAMmSz9Thq --output-file eth_usd_aggregator.bin
+
+# Pyth product: SOL/USD
+solana account 8yrQMUyJRnCJ72NWwMiPV9dNGw465Z8bKUvnUC8P5L6F --output-file 8yrQMUyJRnCJ72NWwMiPV9dNGw465Z8bKUvnUC8P5L6F.bin
+# Pyth price: SOL/USD
+solana account BdgHsXrH1mXqhdosXavYxZgX6bGqTdj5mh2sxDhF8bJy --output-file BdgHsXrH1mXqhdosXavYxZgX6bGqTdj5mh2sxDhF8bJy.bin
+# Pyth product: SRM/USD
+solana account 5agdsn3jogTt8F537GW3g8BuLaBGrg9Q2gPKUNqBV6Dh --output-file 5agdsn3jogTt8F537GW3g8BuLaBGrg9Q2gPKUNqBV6Dh.bin
+# Pyth price: SRM/USD
+solana account 2Mt2wcRXpCAbTRp2VjFqGa8SbJVzjJvyK4Tx7aqbRtBJ --output-file 2Mt2wcRXpCAbTRp2VjFqGa8SbJVzjJvyK4Tx7aqbRtBJ.bin
```

Comment:
Dumps of Pyth accounts for mocking values in the tests. Pyth doesn't currently provide a USDC/USD feed, so we use the SRM account and set the price to $1 later.